### PR TITLE
fix: angular pattern escape problem

### PR DESCRIPTION
**Pull request by jinker**
angular pattern escaped character conversion problem

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the following... 

* Impact:
* Migration path for existing applications: 
* Github Issue(s) this is regarding:

**Other information**:

### CODE REVIEW

**Comment by jinker** [comment]
Have sign CLA. Is the name must github user?

File: index.js [hunk 1]
```diff
@@ -129,6 +129,9 @@ module.exports = function(content) {
 	}
 
 	if(config.interpolate && config.interpolate !== 'require') {
+		content = content.replace(/(ng-)?pattern=(["']?\/((?:.(?!["']?\s+(?:\S+)=|[>"']))+.)\/["']?)/g, function (pattern, $1, $2) {
```
Comment (michael-ciniawsky) [inline index.js]: Would it instead be possible to ignore `ng-*` in general ?

**Comment by michael-ciniawsky** [comment]
@jinker The email address used for your github account and the email address used in `.gitconfig` (`git`) need to match. Otherwise the CLA Bot thinks you are two different entities

File: index.js (hunk 1 above)
Comment (jinker) [reply index.js]: Support ng-* pattern at new commit

**Comment by michael-ciniawsky** [comment]
@jinker Could you please try out the fix proposed in #154 and check if this would resolve your issue ?

**Comment by michael-ciniawsky** [comment]
Closing in favor of #154

**Pull request closed without merging**